aquality-tracking-cucumber5-jvm/src/main/java/aquality/tracking/integrations/cucumber5jvm/TestCaseNameParser.java

```diff
@@ -0,0 +1,51 @@
+package aquality.tracking.integrations.cucumber5jvm;
+
+import io.cucumber.core.internal.gherkin.ast.Feature;
+import io.cucumber.core.internal.gherkin.ast.ScenarioOutline;
+import io.cucumber.core.internal.gherkin.ast.TableRow;
+import io.cucumber.plugin.event.TestCase;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+class TestCaseNameParser {
+
+    private static final String SCENARIO_OUTLINE_KEYWORD = "Scenario Outline";
```

Comment:
can't we fetch this constant from the cucumber package? I think somewhere it is already stored there, but not sure that it is possible to get legal access

Reply:
Unfortunatelly, I didn't it. We can store it on our side because the probability that it changes tends to zero.

Reply:
By the way -"The keyword Scenario Template is a synonym of the keyword Scenario Outline." 
https://cucumber.io/docs/gherkin/reference/
 - do you handle Scenario Template as well?